models/demos/t3000/mixtral8x7b/demo/demo.py

```diff
@@ -320,3 +279,20 @@ def test_mixtral8x7b_demo(t3k_device_mesh, use_program_cache, input_prompts, ins
     return run_mixtral_demo(
         user_input=input_prompts, batch_size=32, device_mesh=t3k_device_mesh, instruct_mode=instruct_weights
     )
+
+
+# CI only runs general-weights demo
+@pytest.mark.skipif(not os.getenv("CI") or os.getenv("CI") == "False", reason="CI-only test")
```

Comment:
is this double condition needed?

Reply:
Not really. I can change to `not os.getenv("CI")=="True"`

Reply:
would we get the same effect with os.getenv("CI")=="False"

if so, lets do that.

Reply:
Unfortunately no. The CI env is not set in non CI machines, so that command will return empty.

But `not os.getenv("CI")=="true"` works fine. Re-running the demo pipeline with these changes to make sure the CI is picking the correct tests only.